### ISSUE TITLE
Pro-Russian spam content in repo

### ISSUE THREAD

**Issue by Vivtek** [moosotc/llpp#191]
The entire repository seems to have been replaced with fascism.

Is @moosotc still around, or has the account been compromised?

**Comment by moosotc** [comment]
Michael Roberts ***@***.***> writes:

> The entire repository seems to have been replaced with fascism.
>
> Is @moosotc still around, or has the account been compromised?
>

I am around. The account is fine and the thing that you took for fascism
adverts is something i've been collecting to document what's happening
over here and, maybe, convince some friends-relatives that it's far from
normal - grotesque, inhumane, and so on - maybe. Some of the video
materials have subtitles so that people who do not speak Russian can get
a taste of how bizare things are over here.

Sorry for inadvertently misleading you.

**Comment by Vivtek** [comment]
I mean, crap, I can't blame you. It's full-on fascism in Russia now (has been for a while). My wife is Hungarian and so we have something like a clue. Everybody back home has gone crazy.

**Comment by moosotc** [comment]
Michael Roberts ***@***.***> writes:

> Ah - you're Ukrainian?

No.

>
> Could you maybe put an explanatory cover page on the project to say
> what you're doing? This really looks like old-school forum spam from
> back in the day.
>

My prose is even worse than my code.

> Is one of the branches appropriate for evaluating llpp?

master with a hard reset to pre-madness (i.e. before 24th of feb) should
get you a working version.

[..]

**Comment by moosotc** [comment]
Michael Roberts ***@***.***> writes:

> Ah. Then my heart really does go out to you. That must be incredibly shitty.

To make sure that we really are on the same page -
   I am _NOT_ Ukrainian. 
>
> Would you like me to write a front page for it? If not, no big deal, I
> get what you're doing.

Yes - i would be very greateful.

**Comment by Vivtek** [comment]
Yeah, I hear you. I won't mention nationalities from here on out.

I'll tread lightly and send you a pull request.
